dynamics-gp/installation/installing-on-subsequent-computers.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: "Client installation"
+title: "Installing Dynamics GP on subsequent computers"
```

Comment:
```suggestion
title: "Installing Dynamics GP on subsequent computers"
```
Per MS Style, we should be using imperative for titles and headings. :)

Reply:
@v-andreaco - it looks this was already done. Thanks!

Reply:
@LJSpiller 
Do you mean in another PR, maybe? In most of the titles that were changed in this PR, the titles are using gerunds when they should be imperative. Have they been changed somewhere else?

Reply:
@v-andreaco - Sorry, I thought you had made an edit. The remaining titles that were not duplicates used gerunds. I matched syntax for the edited titles. But yes, they should be imperatives. I can change all of the titles in another PR.

Reply:
Specifically for this docset, the general style guide does not fully apply. The content is converted from older whitepapers and manuals, written in a different time and age. Fix gerunds to imperative if you like/can automate, but it shouldn't be a hard requirement.

Reply:
Thanks @edupont04!